.travis.yml

```diff
@@ -1,9 +1,23 @@
 language: python
 sudo: false
-python: 2.7
+python:
+    - 2.7
+    - 3.4
+    - 3.5
+    - 3.6
```

Comment:
I do prefer keeping them in just so we can easily see where we stand with future versions at a glance. Also can be helpful for figuring out how much work it would take to add compatibility for certain versions of Django/Python without actually going into the code or running all the tests ourselves.

Reply:
However, I am fine with a compromise of only putting in the 3.6 test for apps which don't already work on Python 3, if that sounds better to you.

Reply:
I don't feel the need for a compromise here. We can do it your way.

Reply:
Heh, had just deleted the comment cause I changed my mind :)

Reply:
but yeah, I kinda like this, I can just look at Travis to see what kind of errors we're up against in future versions.

Reply:
great, now I look like I am passively aggressively insinuating I really did want a compromise, since now I seemingly brought up the idea of a compromise out of nowhere. :roll_eyes:

Reply:
My master plan is working.....

Reply:
I'll never work in this town again :smile: sigh

Reply:
ya know, speaking of compromise :grinning: we could test against all the versions with tox but not in Travis...or do you prefer to see them in Travis?

Reply:
I _could_ do that. I spent many hours trying to get the opposite done, but I suppose I can live with few CI tests and many local tests. I'll go ahead and make the change so we're not waiting 20 mins each commit...